pom.xml

```diff
@@ -103,6 +103,7 @@
         <sonar.dynamicAnalysis>reuseReports</sonar.dynamicAnalysis>
         <sonar.core.codeCoveragePlugin>clover</sonar.core.codeCoveragePlugin>
         <sonar.clover.version>${clover.version}</sonar.clover.version>
+		<maven.deploy.skip>true</maven.deploy.skip>
     </properties>
```

Comment:
Does this change prevent the parent pom from being deployed? It's fine, we don't need to publish the parent pom for now. It looks like there's extra indentation.